### ISSUE TITLE
Implement Tiger Detection in Wild [Object Detection]

### ISSUE THREAD

**Issue by DARK-art108** [CoinShift-Imaging/CoinShift-Imaging-Box#7]
**Detect Tiger from the wild Environment**

Models can be used:
* Faster-RCNN
* SSD
* YoloV3/V4/V5
 
**Dataset:**  https://storage.googleapis.com/openimages/web/index.html [Download From Hereafter filtering]
**Refer to this YT channel for YoloV5 custom Training** 
https://www.youtube.com/channel/UCrydcKaojc44XnuXrfhlV8Q

**Comment by animesh-007** [comment]
@DARK-art108 I would love to work on this with Yolov5. Can you assign me?

**Comment by riyalamba07** [comment]
Hi , I am a GSsoc'21 participant. I would like to work on this issue. Will you please assign this issue to me?

**Comment by DARK-art108** [comment]
Go Ahead @animesh-007 :)

**Comment by DARK-art108** [comment]
> Hi , I am a GSsoc'21 participant. I would like to work on this issue. Will you please assign this issue to me?

You can take a look at other issues. @riyalamba07

**Comment by DARK-art108** [comment]
@animesh-007 what's the status of this? Are you working on it. Any Updates.

**Comment by animesh-007** [comment]
> @animesh-007 what's the status of this? Are you working on it. Any Updates.

Yes. I'm working on this. You can check to see the progress from the colab notebook.
[Colab Link](https://colab.research.google.com/drive/1aIH7xVRIxtQLqr88YCfoZk7QWUcCHeIN?usp=sharing)

**Comment by animesh-007** [comment]
@DARK-art108 My code is ready now. Could you guide me on what should I keep in my PR for this?

- [ ] Should I raise a PR containing only colab notebook?
- [ ] Should I raise a PR containing the code of yolov5?
- [ ] Should I raise a PR containing code and colab notebook both?

**Comment by DARK-art108** [comment]
raise a PR containing code and collab notebook both @animesh-007
